cohort-dashboard/src/pages/Home/components/PostComments/CreateComment/CreateCommentForm.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import CreateCommentBtn from "./CreateCommentBtn";
+
+export default function CreateCommentForm({ postId, setRefreshComments }) {
+  const INITAL_STATE = {
+    postId: postId,
+    content: "",
+    contactId: 1,
+  };
+
+  const [newComment, setNewComment] = useState(INITAL_STATE);
+
+  const createComment = () => {
+    const opts = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(newComment),
+    };
+
+    if (newComment.content) {
+      fetch(
+        `https://boolean-api-server.fly.dev/yee0802/post/${postId}/comment`,
```

Comment:
I would create a file that contains some often re-used constants and put the base api URL in there. You could have a constants.js file that exports a const named API_URL with a value of https://boolean-api-server.fly.dev/yee0802

Reply:
Okay, will do in future. I thought my way was inefficient.